docs/synapse/userguides/storm_ref_pivot.rstorm

```diff
@@ -416,10 +483,34 @@ The pivot out and walk (traverse) light edges operator ( ``--> *`` ) combines a
   
   <query> | tee { --> * } { <-- * }
 
+.. _pivot-out-and-join:
+
+Pivot Out and Join
+------------------
+
+The pivot out and join operator (``--+>``) is similar to the :ref:`pivot-out-and-walk` operator, but joins the results with the inbound set of nodes. That is, the inbound nodes are retained and combined with the results of the edge pivot.
+
+**Syntax:**
+
+*<query>* **----+> ***
```

Comment:
Should be able to escape one of the `-` here to prevent it from turning this into an emdash (or maybe we can disable smartquotes if we don't want that anywhere since it is the one doing that conversion https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-smartquotes)

Reply:
Probably worth fixing in the existing docs too since they currently have an emdash which looks weird.